package.json

```diff
@@ -5,44 +5,38 @@
   "engine": "node 6.11.5",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint --fix .",
-    "react-dev": "webpack -d --watch",
-    "server-dev": "nodemon server/index.js",
+    "festart": "webpack -d --watch",
+    "bestart": "nodemon --inspect server/index.js",
     "seed": "node db/seed.js",
-    "start": "node server/index.js",
     "test": "jest",
     "test:watch": "jest --watch",
+    "lint": "./node_modules/.bin/eslint --fix .",
```

Comment:
This was deleted above and readded here? Why?

Reply:
Line change, so that people could see the more used commands first.